Core/include/Acts/EventData/Measurement.hpp

```diff
@@ -496,10 +497,10 @@ class FixedSizeMeasurement {
   constexpr bool contains(indices_t i) const { return m_subspace.contains(i); }
 
   /// Measured parameters values.
-  const ParametersVector& parameters() const { return m_params; }
+  constexpr const ParametersVector& parameters() const { return m_params; }
 
   /// Measured parameters covariance.
-  const CovarianceMatrix& covariance() const { return m_cov; }
+  constexpr const CovarianceMatrix& covariance() const { return m_cov; }
```

Comment:
I don't think this is actually `constexpr`.

Reply:
`constexpr` removed.